spring-cloud-gcp-samples/spring-cloud-gcp-secretmanager-regional-sample/src/main/java/com/example/SecretManagerRegionalApplication.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2017-2024 the original author or authors.
```

Comment:
For new classes, can you switch to headers like [this one](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/67d994e7f39372be7d0e9010e234f2fe46e68fd9/spring-cloud-gcp-autoconfigure/src/main/java/com/google/cloud/spring/autoconfigure/alloydb/AlloyDbProperties.java#L1-L15), with 2025 for the year? We have this header discrepancy due to historical migration reasons. Sorry for the confusion. 

```suggestion
 * Copyright 2025 Google LLC
```

Reply:
No worries. Updated